query/language.peg

```diff
@@ -105,7 +105,7 @@ expression_function  <-
     p.addStringLiteral(unescapeLiteral(buffer[begin:end]))
   }
   PAREN_OPEN
-    expressionList (__ groupByClause)?
+    expressionList (__ groupByClause / { p.addGroupBy() } )
```

Comment:
Since the addGroupby() is common in both paths, can we make this `expressionList { p.addGroupBy() } (__ groupByClause)?`, and pull the addGroupBy out of groupByClause?

It also looks like the `group by` in a function is going to apply over all arguments?